components/image-placeholder/ImageWithPlaceholder.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+import React from 'react';
+import { PostImagePlaceholder } from '@/components/placeholders/PostImagePlaceholder';
+import Image from 'next/image';
+
+interface IProps {
+  src: string;
+  alt: string;
+}
+
+const ImageWithPlaceholder = ({ src, alt }: IProps) => {
+  const [loaded, setLoaded] = React.useState(false);
+  const imageProportions = 'pb-[calc((8/17)*100%)]';
```

Comment:
If the value `calc((8/17)*100%)` is used in several components, we could also use a global css class as we did for `global-width`